components/MenuEditorials/index.js

```diff
@@ -26,9 +26,9 @@ const MenuEditorials = ({amp, color, items, title, titleProps, groupTitleProps,
       }
       {items && 
         map(items, (item, key) => amp ? 
-          <AmpMenuItem color={color} content={item} groupTitleProps={groupTitleProps} key={key} removeBorders={removeBorders} />
+          <AmpMenuItem color={color} content={item} groupTitleProps={groupTitleProps} groupSubItemProps={groupSubItemProps} key={key} removeBorders={removeBorders} />
```

Comment:
Acho que está faltando aqui e no MenuItem a conexão da prop `pl` que foi passada lá dentro. Se não ela será uma deadProp (nunca poderá receber um dado do front).

Derrepente podemos criar um `menuItemProps` que nem fizemos com o Title e SubItem. Dentro desse novo conjunto de props poderia ser passado a nova prop pl.

No fim das contas fica algo mais ou menos assim
```suggestion
          <AmpMenuItem color={color} content={item} groupTitleProps={groupTitleProps} groupSubItemProps={groupSubItemProps} key={key} removeBorders={removeBorders} {...menuItemProps} />
```